### ISSUE TITLE
Is RunSuite.txt obsolete?

### ISSUE THREAD

**Issue by CyberiaResurrection** [makhidkarun/traveller_pyroute#47]

**Comment by tjoneslo** [comment]
Yes. it almost certainly is. I've been using it to keep track of command lines to run for execution of different data sets. But since adding the `@<file>` the file could be replaced with a number of individual files to execute the same.
